ha/k8s_setup/ha_setup.py

```diff
@@ -264,6 +264,8 @@ def process(self):
 
             Log.info(f'Populating the ha config file with consul_endpoint: {consul_endpoint}')
 
+            Log.info('Performing event_manager subscription')
+            event_manager = EventManager.get_instance()
```

Comment:
Why it is showing as a new change? I see that its already there!

Reply:
Rebased.

Reply:
Ok..